rtdata/languages/default

```diff
@@ -4331,6 +4383,11 @@ TP_WAVELET_OPACITYWL_TOOLTIP;Modify the final local contrast at the end of the w
 TP_WAVELET_PASTEL;Pastel chroma
 TP_WAVELET_PROC;Process
 TP_WAVELET_PROTAB;Protection
+TP_WAVELET_PYRWAVTRC;Contrast profile
+TP_WAVELET_PYRWAVTRC_TOOLTIP;The 'Contrast profile' slider lets you select a series of presets that will significantly modify the effect of the “Variable contrast” curve. The presets range from least to most aggressive in ascending order.\n\nFor the upper preset levels, the visibility of the result in the preview window will depend on the image size and the system configuration. Increase the preview window size if this is the case.
```

Comment:
I don't know what to answer, we are doing semantics, in English. Both are correct. But, detail, because you grouped the tooltips into one for the title, nothing appears in RT anymore.

Another "detail" is that the French and English languages ​​do not use the same syntax and the same conventions (we talked about this at length with Wayne). In addition, it is not easy to transcribe what is essential and here again, Latins and Anglo-Saxons do not seem to reason in the same way.
:)